app/assets/javascripts/smart-answers.js

```diff
@@ -27,7 +26,6 @@ $(document).ready(function() {
 
     // Track when a user clicks on 'Start again' link
     $('.start-right').live('click', function() {
-      window._gaq && window._gaq.push(['_trackEvent', 'MS_smart_answer', getCurrentPosition(), 'Start again']);
```

Comment:
Is the `getCurrentPosition` function used by anything else? If not that can also be removed.